benchmarks/pipe.py

```diff
@@ -43,7 +44,14 @@ def init_random_seed(seed: int):
     numpy.random.seed(seed)
 
 
-def make_model(args, device, config):
+def get_model_and_optimizer(args, device, config):
+
+    if args.model_name == "seq_pred":
+        return get_seq_pred_model(args, device, config)
+
+
+def get_seq_pred_model(args, device, config):
```

Comment:
docstring.

this model is usually called LM (language model). perhaps rename the string to "lm" or "language_model" and rename the function get_language_model().

Also, we can use python enum instead of string in the place of "seq_pred".

Reply:
I modified the name since we will have a second benchmark which is also a language model. The two names that I had in mind were "seq_pred" (existing model) and "seq_trans" (new model). Language model seemed too generic given the two benchmarks we want to add. Open to other naming suggestions if the previously mentioned ones don't feel right.

I am not convinced about if we should add an enum. Implementing a flag based enum will be a bit of boilerplate code. LMK if you feel strongly and I can add the implementation.

Reply:
Usually, seq_pred is called LM, seq_trans is called NMT. Language modeling (not language model) is a special kind of task. So, I would recommend LM and NMT.

String is fine, since you have argparse's choice to select the right string anyway.

Reply:
Done.